app/build.gradle

```diff
@@ -190,6 +193,7 @@ dependencies {
 
     kapt "com.jakewharton:butterknife-compiler:${deps.butterknife}"
     kapt "org.greenrobot:eventbus-annotation-processor:${deps.eventbus}"
+    compile project(path: ':ui:tutorial-renderer')
```

Comment:
starting a new thread about this since the previous comment seems to have been lost.

this dependency should be unnecessary because you already define it earlier